### ISSUE TITLE
E:\kohya_ss\venv\lib\site-packages\torch\utils\checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None   warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")

### ISSUE THREAD

**Issue by CRCODE22** [bmaltais/kohya_ss#1736]
When starting to train a lora I am getting the following error:

E:\kohya_ss\venv\lib\site-packages\torch\utils\checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")

use AdamW optimizer | {}
running training / 学習開始
num train images * repeats / 学習画像の数×繰り返し回数: 6720
num reg images / 正則化画像の数: 0
num batches per epoch / 1epochのバッチ数: 1680
num epochs / epoch数: 16
batch size per device / バッチサイズ: 4
gradient accumulation steps / 勾配を合計するステップ数 = 1
total optimization steps / 学習ステップ数: 26880
steps: 0%| | 0/26880 [00:00<?, ?it/s]
epoch 1/16
E:\kohya_ss\venv\lib\site-packages\torch\utils\checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")

I am using the latest version of Kohya_ss as it updated not long ago
pip install -r requirements.txt
Then I ran the update process just to be sure
Error still occurs.
